arch/aarch64/mmu.c

```diff
@@ -2,6 +2,12 @@
 
 #include <kernel/errno.h>
 
+/////////////////////////////////////////////////////////////////////////////
+// \brief method converts address from virtual to physical memory
+//
+// \param accepts an address variable
```

Comment:
The correct format for the `param` tag is:

```
\param <name of parameter> <description of parameter>
```

So, for example, here it should be:

```
\param addr Virtual address to translate.
```

Reply:
Right, realising didn't properly work in adding Doxygen documentation. I will give another pass.